tests/ballerina-test-utils/src/main/java/org/ballerinalang/test/util/HttpClientRequest.java

```diff
@@ -116,13 +116,10 @@ public static HttpResponse doPost(String endpoint, String postBody, Map<String,
         try {
             urlConnection = getURLConnection(endpoint);
             setHeadersAndMethod(urlConnection, headers, TestConstant.HTTP_METHOD_POST);
-            OutputStream out = urlConnection.getOutputStream();
-            try {
+            try (OutputStream out = urlConnection.getOutputStream()) {
```

Comment:
We can include writer as well into the try block. Will send the changes separately